crates/sui-framework/packages/sui-system/sources/validator.move

```diff
@@ -221,11 +221,14 @@ module sui_system::validator {
         ctx: &mut TxContext
     ): Validator {
         assert!(
-            // MUSTFIX: These constants are arbitrary, will adjust once we know more.
             vector::length(&net_address) <= 128
                 && vector::length(&p2p_address) <= 128
-                && vector::length(&name) <= 128
-                && vector::length(&description) <= 150,
+                && vector::length(&primary_address) <= 128
+                && vector::length(&worker_address) <= 128
+                && vector::length(&name) <= 80
+                && vector::length(&description) <= 256
+                && vector::length(&image_url) <= 128
+                && vector::length(&project_url) <= 128,
             0
```

Comment:
There are some checks in genesis.rs that should be updated to reflect this. Also are we sure these are the values we want for these? I'd be more inclined to have larger limits (256 for each maybe?)

Reply:
Specifically here: https://github.com/MystenLabs/sui/blob/main/crates/sui-config/src/genesis.rs#L341

Reply:
We should unify these checks...in the near future.

Reply:
will update `genesis.rs`. I don't have opinions on what limits to use but 256 for everything sounds reasonable to me

Reply:
updated genesis.rs and added checks in the metadata rotation path